--- "ESP8266_RTOS_SDK-master\\driver_lib\\driver\\i2c_master.c.orig"	2017-06-11 13:04:26.168175300 +0900
+++ "ESP8266_RTOS_SDK-master\\driver_lib\\driver\\i2c_master.c"	2017-06-11 13:09:28.190190600 +0900
@@ -105,8 +105,8 @@
 void ICACHE_FLASH_ATTR
 i2c_master_gpio_init(void)
 {
-    ETS_GPIO_INTR_DISABLE() ;
-//    ETS_INTR_LOCK();
+//    ETS_GPIO_INTR_DISABLE() ;
+    ETS_INTR_LOCK();
 
     PIN_FUNC_SELECT(I2C_MASTER_SDA_MUX, I2C_MASTER_SDA_FUNC);
     PIN_FUNC_SELECT(I2C_MASTER_SCL_MUX, I2C_MASTER_SCL_FUNC);
@@ -118,8 +118,8 @@
 
     I2C_MASTER_SDA_HIGH_SCL_HIGH();
 
-    ETS_GPIO_INTR_ENABLE() ;
-//    ETS_INTR_UNLOCK();
+//    ETS_GPIO_INTR_ENABLE() ;
+    ETS_INTR_UNLOCK();
 
     i2c_master_init();
 }
